### PR TITLE
[RFC]: chore: move integration tests to examples/ folder

### PR DESCRIPTION
**Pull request by ZauberNerd**

### CODE REVIEW

File: examples/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
```
Comment (herschel666) [inline examples/package.json]: what purpose does this `package.json` serve?

Comment (ZauberNerd) [reply examples/package.json]: As far as I know it is needed because this is also a package in the workspace.

Comment (herschel666) [reply examples/package.json]: only its contents should be packages. `./packages` doesn't have a `package.json`.

Comment (ZauberNerd) [reply examples/package.json]: You are right, we could get away without this `package.json`, but if we want to lint the examples, we need to put the eslint config into our global config, which would make our global config less strict and we also have to ignore some eslint rules..
See this commit: https://github.com/xing/hops/pull/718/commits/ce2e293590e8d9b4c58ba6b4a2809899c6e0b383

Comment (herschel666) [reply examples/package.json]: What about about "outsourcing" the config into `.eslintrc.js` files and have the `/examples` one extend the one in the root of the project?

**Comment by ZauberNerd** [comment]
Closing this for now. Maybe we will revisit this topic later when we have more time.

**Pull request closed without merging**